README.md

```diff
@@ -523,9 +523,10 @@ module.exports = {
 
 By defining/modifying `networks["integratedDevnet"]` in your hardhat config file, you can specify:
 
--   the version of Devnet to be used for the underlying Devnet Docker image
--   a Python environment with installed starknet-devnet (can be active environment); this will avoid using the dockerized version
--   CLI arguments to be used on Devnet startup: [options](https://github.com/Shard-Labs/starknet-devnet/#run)
+- the version of Devnet to be used for the underlying Devnet Docker image
```

Comment:
What happened to indentation here?

Reply:
I see that md warns with [MD0303](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md030) when I've more than one space after a list item. But I've restored it. https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/179/commits/0f2771e75bd3a74fcbea419882111fe1a7c4ca3e

Reply:
Not saying that I prefer the old version with multiple spaces, that's just what my formatter gives me when I apply it